app.py

```diff
@@ -32,6 +35,13 @@
 
 app = Flask(__name__)
 
+log_handler = CloudWatchLogHandler(
+    log_group_name=f"citybase_{flask_env}", log_stream_name="postback_stream"
```

Comment:
added the env variable to know which environment we are in so that the logs are different per version being run

Reply:
What do you think about making this "development", or changing the docker compose profile name to "local"? I was getting a little confused by the profile name versus the log group names.

What happens if you set the `flask_env` to `"pizza"`? Would that create a new log group of that name? It'd be good to document this as well.

I am also seeing we still have this line down at the bottom of the this file:


```python
if __name__ == "__main__":
    # todo: remember to turn off debug!
    app.run(debug=True, host="0.0.0.0")
```

Can we make use of the `flask_env` here so that we don't need to remember anything?

Reply:
I am wondering if the `log_stream_name` should update daily, to break them up and make them be easier to search in the future, instead of one monster one. 

`log_stream_name=f"postback_stream_{datetime.now().strftime("%Y%m%d")}"`